test/commons/text/accessible-text.js

```diff
@@ -257,12 +257,12 @@ describe('text.accessibleTextVirtual', function() {
 		axe.testUtils.flatTreeSetup(fixture);
 
 		var target = axe.utils.querySelectorAll(axe._tree, '#t2label')[0];
-		// Chrome 72: This is This is a label of
-		// Firefox 62: This is ARIA Label
-		// Safari 12.0: This is This is a label of
+		// Chrome 86: This is This is a label of
```

Comment:
Are we marking what the browser's accessibility tree says or what screen readers with those browsers say?

Reply:
the browser's name in the acc tree.